libfwupdplugin/fwupdplugin.map

```diff
@@ -1067,3 +1067,18 @@ LIBFWUPDPLUGIN_1.8.3 {
     fu_memread_uint24_safe;
   local: *;
 } LIBFWUPDPLUGIN_1.8.2;
+
+LIBFWUPDPLUGIN_1.8.4 {
+  global:
+    fu_bios_attributes_get_attr;
+    fu_bios_attributes_get_pending;
+    fu_bios_attrs_get_all;
+    fu_bios_attrs_get_type;
+    fu_bios_attrs_new;
+    fu_bios_attrs_reinitialize;
+    fu_context_get_firmware_attr;
+    fu_context_get_firmware_attr_pending;
+    fu_context_get_firmware_attrs;
+    fu_context_reload_firmware_attributes;
```

Comment:
fu_context_reload_firmware_attrs for consistency perhaps?

Reply:
Actually, this should probably be fu_context_get_bios_*

Reply:
with how it's used right now (for self tests) `fu_context_reload_firmware_attrs` makes most sense.
I do think there is opportunity later for reacting to drivers being loaded/unloaded from the kernel and connecting to a signal for that too for the daemon.